zerver/test_bugdown.py

```diff
@@ -175,6 +175,11 @@ def replaced(payload, url, phrase=''):
             converted = bugdown_convert(inline_url)
             self.assertEqual(match, converted)
 
+    def test_inline_file(self):
+        msg = 'Check out this file file:///Volumes/myserver/Users/Shared/pi.py'
+        converted = bugdown_convert(msg)
+        self.assertEqual(converted, '<p>Check out this file <a href="file:///Volumes/myserver/Users/Shared/pi.py" target="_blank" title="file:///Volumes/myserver/Users/Shared/pi.py">file:///Volumes/myserver/Users/Shared/pi.py</a></p>')
+
```

Comment:
I think it'd be good to have tests for correct behavior with both `ENABLE_FILE_LINKS=False` and `ENABLE_FILE_LINKS=True` 

https://docs.djangoproject.com/en/dev/topics/testing/tools/#overriding-settings is probably relevant for how to write such tests.